src/plugins/visualizations/public/services.ts

```diff
@@ -66,3 +67,6 @@ export const [getOverlays, setOverlays] = createGetterSetter<OverlayStart>('Over
 export const [getChrome, setChrome] = createGetterSetter<ChromeStart>('Chrome');
 
 export const [getSpaces, setSpaces] = createGetterSetter<SpacesPluginStart>('Spaces', false);
+
+export const [getVisEditorsRegistry, setVisEditorsRegistry] =
```

Comment:
Not sure that we need to add one more `getterSetter` for that case. I've checked all places where we use it and looks like you can just add this service into 

```
src/plugins/visualizations/public/plugin.ts  
const services: VisualizeServices = {
```

Could you please check that?